consul/base.py

```diff
@@ -238,6 +238,10 @@ def __init__(
         # TODO: Status
 
         if os.getenv('CONSUL_HTTP_ADDR'):
+            # Checking if the environment variable contains exactly one ':'
+            colon_count = os.getenv('CONSUL_HTTP_ADDR').count(':')
+            assert colon_count == 1, 'The environment variable CONSUL_HTTP_ADDR is not setup according to the standards.'
+            # Splitting the host and port based on the colon
```

Comment:
I wonder if a better approach would be to parse CONSUL_HTTP_ADDR robustly, and then extract host, port, instead of relying on the naive split. 

https://docs.python.org/2/library/urlparse.html#urlparse.urlparse

The example have in #149 would then just work.

Care just needs to be taken so it works for python 2 + 3..